scripts/config.js

```diff
@@ -0,0 +1,398 @@
+        return 'Please enter Stripe publishable key with prefix pk_!';
+      },
+      ...stirpeDefault,
+    },
+    {
+      type: 'input',
+      name: 'REACT_APP_MAPBOX_ACCESS_TOKEN',
+      message: `What is your Mapbox access token?
+${chalk.dim(
+        `Mapbox is the default map provider of the application. Sign up for Mapbox and go the to account page. Then click Create access token. For more information see the: Integrating to map providers documentation.
+If you don't set the Mapbox key, the map components won't work in the application.`
+      )}
+`,
+      ...mapBoxDefault,
+    },
+    {
+      type: 'input',
+      name: 'REACT_APP_SHARETRIBE_MARKETPLACE_CURRENCY',
+      message: `What is your marketplace currency?
+${chalk.dim(
+        'The currency used in the Marketplace must be in ISO 4217 currency code. For example USD, EUR, CAD, AUD, etc. The default value is USD.'
+      )} 
+`,
+      default: function() {
+        return currencyDefault ? currencyDefault : 'USD';
+      },
+      validate: function(value) {
+        if (value.match(/^[a-zA-Z]{3}$/)) {
+          return true;
+        }
+        return 'Please enter currency in ISO 4217 format (e.g. USD, EUR, CAD...)';
+      },
+    },
+  ];
+};
+
+/**
+ * Questions for advances variables. User can choose to edit these or not.
+ * If some values already exist in the .env file use them as a default.
+ * Otherwise don't use default value in the questions.
+ *
+ * @param {array} settings - array of values used as default if user is editing the .env file
+ *
+ * @returns array of questions passed to Inquirer
+ *
+ */
+
+const advancedSettings = settings => {
+  const rootUrlDefault = settings ? settings.REACT_APP_CANONICAL_ROOT_URL : null;
+  const availabilityDefault = settings
+    ? settings.REACT_APP_AVAILABILITY_ENABLED.toLowerCase() === 'true'
+    : true;
+  const searchesDefault = settings
+    ? settings.REACT_APP_DEFAULT_SEARCHES_ENABLED.toLowerCase() === 'true'
+    : true;
+
+  return [
+    {
+      type: 'confirm',
+      name: 'showAdvancedSettings',
+      message: 'Do you want to edit advanced settings?',
+      default: false,
+    },
+    {
+      type: 'input',
+      name: 'REACT_APP_CANONICAL_ROOT_URL',
+      message: `What is your canonical root URL? 
+${chalk.dim(
+        'Canonical root URL of the marketplace is needed for social media sharing and SEO optimization. When developing the template application locally URL is usually http://localhost:3000'
+      )}
+`,
+      default: function() {
+        return rootUrlDefault ? rootUrlDefault : 'http://localhost:3000';
+      },
+      when: function(answers) {
+        return answers.showAdvancedSettings;
+      },
+    },
+    {
+      type: 'confirm',
+      name: 'REACT_APP_AVAILABILITY_ENABLED',
+      message: `Do you want to enable availability calendar?
+${chalk.dim(
+        'This setting enables the Availability Calendar for listings. The default value for this setting is true.'
```

Comment:
@OtterleyW sorry, I have completely forgotten that this environment variable exists. So, there's a bug - not a critical one, but definitely this wording doesn't make sense atm. My bad.

Before merging this PR, could you add a bug-fix to availability calendar code - it could be a part of this PR or added as a separate one.

What should be done:

**config.js** (expose new config):
```js
// Show availability calendar on EditListingWizard
const showAvailabilityCalendar = process.env.REACT_APP_AVAILABILITY_ENABLED === 'true';
```
**+** exposing it at the end of the file

**EditListingWizard.js** (conditionally show tab):
```js
import classNames from 'classnames';
import config from '../../config';
```
```js
const availabilityMaybe = config.showAvailabilityCalendar ? [AVAILABILITY] : [];
export const TABS = [DESCRIPTION, FEATURES, POLICY, LOCATION, PRICING, ...availabilityMaybe, PHOTOS];
```

**ManageListingCard.js** (conditionally show "manage listing" link):
```js
{config.showAvailabilityCalendar
  ? (
    <React.Fragment>
      <span className={css.manageLinksSeparator}>{' • '}</span>

      <NamedLink
        className={css.manageLink}
        name="EditListingPage"
        params={{ id, slug, type: editListingLinkType, tab: 'availability' }}
      >
        <FormattedMessage id="ManageListingCard.manageAvailability" />
      </NamedLink>
    </React.Fragment>
    )
  : null}
```